src/headers/header_value.rs

```diff
@@ -11,7 +12,7 @@ use crate::Mime;
 /// A header value.
 #[derive(Clone, Eq, PartialEq, Hash)]
 pub struct HeaderValue {
-    inner: String,
+    inner: Cow<'static, str>,
```

Comment:
why not `pub struct HeaderValue<'a> { inner: Cow<'a, str> }` and just use `'static` where we need it to be static?

Reply:
@jbr That'd make the API a little more awkward to use, and would potentially be a backwards-compatibility issue.